repos/system_upgrade/el7toel8/actors/rpmtransactionconfigtaskscollector/libraries/scanner.py

```diff
@@ -19,7 +20,20 @@ def load_tasks_file(path, logger):
 
 def load_tasks(base_dir, logger):
     # Loads configuration files to_install, to_keep, and to_remove from the given base directory
+    rpms = next(api.consume(InstalledRedHatSignedRPM))
+    rpm_names = [rpm.name for rpm in rpms.items]
+    to_install = load_tasks_file(os.path.join(base_dir, 'to_install'), logger)
+    # we do not want to put into rpm transaction what is already installed (it will go to "to_upgrade" bucket)
+    to_install_filtered = [pkg for pkg in to_install if pkg not in rpm_names]
+
+    filtered = set(to_install) - set(to_install_filtered)
+    if filtered:
+        api.current_logger().debug(
+            'The following packages from "to_install" file  will be ignored as they are already installed:'
+            '\n- ' + '\n- '.join(filtered))
+
+
```

Comment:
It didn't go through pylint. Too many blank lines.

Reply:
ok, let's satisfy the linter